lib/takeaway.rb

```diff
@@ -0,0 +1,33 @@
+class Restaurant
+    attr_reader :menu, :order
+
+    def initialize
+        @menu = { 'burger': 5,
+                  'pizza': 6,
+                  'lasagna': 4,
+                  'ice_cream': 2
+                }
+        @order = []
+    end 
+
+    def menu_print
+        @menu.each { |key, value| puts "#{key}: £#{value}" }
+    end 
+
+    def add_item(item_from_menu)
+        @order << item_from_menu
```

Comment:
The user wants to be able to add multiple of the same item at the same time. That is not possible at the moment. How could you do this? Anything they pass will go onto the order. It doesn't matter what it is. This will break your code later when they try to total the order as they are not inputing a hash with an integer assigned as the value. Given more time it is important to test for edge cases.